start_stop.lua

```diff
@@ -472,6 +456,11 @@ function courseplay:start(self)
 		-- Initialize pure pursuit controller
 		self.cp.ppc = PurePursuitController(self)
 		self.cp.ppc:initialize()
+		if self.cp.driver ~= nil and self.cp.driver.collisionDetector ~= nil then
+			self.cp.driver.collisionDetector:deleteTriggers()
+		end
+		local ret_createLegacyCollisionTriggers = false
```

Comment:
why the variable?

Reply:
See next line - it gets the return value from the function call.
In general I think it is a good approach to get a feedback from a function if it was successful or not.
This could be used for further processing / error handling / information to the user...
I just want to make the developer aware that there is a result provided from the function, not only fire&forget

Reply:
See above.